.github/workflows/kitchen.yml

```diff
@@ -13,45 +13,91 @@ jobs:
+          [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12;
+          $package_destination = "$env:temp\rubyinstaller-devkit-$pkg_version-$pkg_revision-x64.exe"
+          (New-Object System.Net.WebClient).DownloadFile($pkg_source, $package_destination);
+          Write-Output "Did the file download?"
+          $output = Get-ChildItem -Path $env:temp
+          Write-Output $output
+          Write-Output 'Installing Ruby + DevKit';
+          Start-Process $package_destination -ArgumentList '/verysilent /dir=C:\ruby31' -Wait ;
+          Write-Output 'Cleaning up installation';
+          Remove-Item $package_destination -Force;
+          Write-Output "Installing URU to manage Ruby Versions"
+          choco install 7zip -y
+          Write-Output "Downloading Uru Installer..."
+          New-Item -Path c:\uru_temp -Type Directory
+          # Use TLS 1.2 for Windows 2016 Server and older
+          [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+          Invoke-WebRequest -OutFile "c:\uru_temp\uru-0.8.5-windows-x86.7z" -Uri "https://bitbucket.org/jonforums/uru/downloads/uru-0.8.5-windows-x86.7z"
+          Write-Output "Installing Uru Ruby Switcher"
+          7z x "c:\uru_temp\uru-0.8.5-windows-x86.7z" -o"C:\Program Files (x86)\Uru"
+          If ($lastexitcode -ne 0) { Exit $lastexitcode }
+          Start-Process "C:\Program Files (x86)\Uru\uru_rt.exe" -WorkingDirectory "C:\Program Files (x86)\Uru\" -ArgumentList 'admin install' -Wait
+
+          Write-Output 'Updating PATH'
+          $env:PATH = "C:\Program Files (x86)\Uru;" + $env:PATH
+          [Environment]::SetEnvironmentVariable('PATH', $env:PATH, [EnvironmentVariableTarget]::Machine)
+
+          Write-Output "Register Installed Ruby Version 3.1 With Uru"
+          Start-Process "uru_rt.exe" -ArgumentList 'admin add C:\ruby31\bin' -Wait
+          uru 312
+          if (-not $?) { throw "Can't Activate Ruby. Did Uru Registration Succeed?" }
+          ruby -v
+          if (-not $?) { throw "Can't run Ruby. Is it installed?" }
+        }
+
     - name: 'Install Chef/Ohai from Omnitruck'
       id: install_chef
       run: |
         . { Invoke-WebRequest -useb https://omnitruck.chef.io/install.ps1 } | Invoke-Expression; Install-Project -project chef -channel current
-        $env:PATH = "C:\opscode\chef\bin;C:\opscode\chef\embedded\bin;" + $env:PATH
+        $env:PATH = "C:\ruby31\bin;C:\opscode\chef\bin;C:\opscode\chef\embedded\bin;" + $env:PATH
         chef-client -v
         ohai -v
         rake --version
         bundle -v
-    - name: 'Upgrade Chef/Ohai via Appbundler'
-      id: upgrade
-      run: |
-        $env:PATH = "C:\opscode\chef\bin;C:\opscode\chef\embedded\bin;" + $env:PATH
-        $env:OHAI_VERSION = ( Select-String -Path .\Gemfile.lock -Pattern '(?<=ohai \()\d.*(?=\))' | ForEach-Object { $_.Matches[0].Value } )
+    # - name: 'Upgrade Chef/Ohai via Appbundler'
```

Comment:
Is this a candidate for removal or a temporary comment?

Reply:
temporary I think to get the first build out then we would like fall back to using the previous bits